index.html

```diff
@@ -557,82 +442,91 @@ <h3>
       </section>
       <section>
         <h3>
-          <dfn>request()</dfn> method
+          <dfn>request()</dfn> static method
         </h3>
-        <p>
-          To <dfn>abort the request steps</dfn> for a running instance of
-          <a data-link-for="WakeLock">request()</a>, run these additional
-          steps:
-        </p>
-        <ol class="algorithm" data-link-for="WakeLock">
-          <li>Set |lock:WakeLock|.{{[[requestPromise]]}} to `undefined`.
-          </li>
-          <li>Abort the associated <a>request()</a> algorithm.
-          </li>
-        </ol>
         <p data-tests="wakelock-api.https.html" data-link-for="WakeLock">
-          The <a>request()</a> method, when invoked, MUST run the following
-          steps:
+          The <a data-lt="request">request(|type:WakeLockType|,
+          |options:WakeLockRequestOptions|)</a> method, when invoked,
+          MUST run the following steps:
         </p>
         <ol class="algorithm">
-          <li>Let |options:WakeLockRequestOptions| be the first argument.
-          </li>
-          <li>Let |lock:WakeLock| be the <a>context object</a>.
-          </li>
           <li>Let |promise:Promise| be <a data-cite=
-            "promises-guide#a-new-promise">a new promise</a>.
-          </li>
-          <li>If |lock|.{{[[requestPromise]]}} is not `undefined`,
-            reject |promise| with a "{{InvalidStateError}}" {{DOMException}},
-            and run <a>abort the request steps</a>.
+          "promises-guide#a-new-promise">a new promise</a>.
           </li>
-          <li>If |options|.|signal:AbortSignal|’s <a>aborted flag</a> is set, then
-          reject |promise| with an "{{AbortError}}" {{DOMException}},
-          and run <a>abort the request steps</a>.
+          <li>If |options|.|signal:AbortSignal| member is present, then:
```

Comment:
As Marcos mentioned elsewhere `|foo|.|bar|` doesn't make sense because it defines a new variable `bar`. Instead we need language like, "Let |bar| be |foo|'s "bar" member."

Reply:
I agree, and there might be more of these. It was in the original code after all and not a change by this PR.

I am travelling now until the 23rd and quite busy. Do you mind if we land this and have @rakuco do a few follow up patches?

Reply:
I'm okay with merging this as is and continuing refinement in later PRs. Let's make sure to leave an issue open to track that work.

Reply:
@kenchris has filed #208 to track this